style.css

```diff
@@ -303,7 +326,7 @@ button:disabled {
   margin-left: 1rem;
   font-weight: 500;
   font-style: normal;
-  width: 60%;
+  width: 50%;
```

Comment:
- [ ] kindly use the correct color of `#344563` for the `strong` style in line 343 to 349 for your language, framework and skills